Formula/geant4.rb

```diff
@@ -14,70 +14,17 @@ class Geant4 < Formula
   end
 
   depends_on "cmake" => [:build, :test]
+  depends_on "expat"
```

Comment:
Why add?

Reply:
add this line geant4 would use the expat homebrew version, or it would use the expat system version, this may cause some bug if using geant4 and other software together

Reply:
Perhaps `uses_from_macos "expat"` would be appropriate here

Reply:
I prefer using `depends_on` instead of `uses_from_macos`, it support using expat library from pkgconfig.